docker-compose.yml

```diff
@@ -19,6 +19,8 @@ services:
     build:
       context: .
       dockerfile: docker/Dockerfile.viewer
+      args:
+        PI_VERSION: ${DEVICE_TYPE}
```

Comment:
No, we can't do this. `docker-compose.yml` is reserved for Balena, and they don't support arguments in their `docker-compose.yml` file.

Reply:
How about a [label](https://docs.docker.com/engine/reference/builder/#label)?